### PR TITLE
[QuickMenu] Support dialog rotation

### PR DESCRIPTION
**Pull request by codeimpl**
When dialog is rotating, child view on dailog was broken. So the dialog
adds RotationManager to get notify of system orientation change.

Refs: #1007

### CODE REVIEW

File: src/org/triple/banana/quick_menu/QuickMenuDialog.java [hunk 1]
```diff
@@ -89,17 +98,47 @@ private void setColumnWeight(@NonNull QuickMenuMiddleButton button) {
         button.setLayoutParams(params);
     }
 
+    private void changeColumnCount(int columnCount) {
```
Comment (romandev) [inline src/org/triple/banana/quick_menu/QuickMenuDialog.java:101]: I think the better name might be `relayout`

File: src/org/triple/banana/quick_menu/QuickMenuDialog.java [hunk 2]
```diff
@@ -89,17 +98,47 @@ private void setColumnWeight(@NonNull QuickMenuMiddleButton button) {
...
+    @Override
+    public void dismiss() {
+        super.dismiss();
+
+        mRotationManager.removeListener(mOrientationChangedListener);
+    }
+
+    private RotationManager.Listener createOrientationChangedListener() {
+        return info -> {
+            mHandler.post(() -> setGridLayoutColumn());
```
Comment (romandev) [inline src/org/triple/banana/quick_menu/QuickMenuDialog.java:141]: FYI, the `info` has already the orientation information. It means that the `setGridLayoutColumn()` might be unnecessary.

File: src/org/triple/banana/quick_menu/QuickMenuDialog.java (hunk 1 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuDialog.java]: sure, thanks. done!

File: src/org/triple/banana/quick_menu/QuickMenuDialog.java [hunk 3]
```diff
@@ -89,17 +98,43 @@ private void setColumnWeight(@NonNull QuickMenuMiddleButton button) {
...
+
+        mRotationManager.removeListener(mOrientationChangedListener);
+    }
+
+    private RotationManager.Listener createOrientationChangedListener() {
+        return info -> {
+            mHandler.post(() -> {
+                relayout(info.orientation == RotationManager.Orientation.LANDSCAPE
+                                ? Configuration.ORIENTATION_LANDSCAPE
+                                : Configuration.ORIENTATION_PORTRAIT);
```
Comment (romandev) [inline src/org/triple/banana/quick_menu/QuickMenuDialog.java:136]: There is no reason to convert the `RotationManager.Orientation.XXX` to `Configuration.ORIENTATION_XXX`

Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuDialog.java:136]: relayout function is called by onContentChanged() and here. But inside the onContentChanged only know orientatino of Configuration. So I thinks that it needs converting.

Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuDialog.java]: @romandev Could you check my code again?? thanks!

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**